.appveyor.yml

```diff
@@ -35,4 +35,8 @@ before_build:
   - cd C:\projects\metabench
 
 build_script:
-  - bash tools\run_tests.sh -DRUBY_LIBRARY=%RUBY_LIBRARY%
+  - mkdir build
+  - cd build
+  - cmake .. -G%CMAKE_GENERATOR% -DRUBY_LIBRARY=%RUBY_LIBRARY% -DMETABENCH_UNIT_TEST_OPTIONS="-DRUBY_LIBRARY=%RUBY_LIBRARY%"
```

Comment:
It's super annoying. For some reason, the library can't be found (whether in path or not). I think it's a bug in the FindRuby module (like a name mismatch). At some point I just figured I could set `RUBY_LIBRARY` to shut it up, but I agree that we should fix the root of the problem. I'm looking into it right now.